db/migrations/20230227122704-3-ss58-prefix.js

```diff
@@ -0,0 +1,249 @@
+        }
+
+        queryString += ` WHERE rank='${rank}'`;
+
+        await executeDbRunSqlAsPromise(db, queryString);
+
+        logger.log();
+    }
+}
+
+const convertFaucetTable = async (db, ss58Format) => {
+    const logger = new Logger('faucet');
+
+    const result = await executeDbRunSqlAsPromise(db, 'SELECT id, sender, destination from faucet;');
+    const rows = result.rows;
+    logger.setTotalRows(rows.length);
+
+    for (let i = 0; i < rows.length; i++) {
+        const {id, sender, destination} = rows[i];
+
+        const nextSender = decode(sender, ss58Format);
+
+        const nextDestination = decode(destination, ss58Format);
+
+        const queryString = `UPDATE faucet SET sender='${nextSender}', destination='${nextDestination}' WHERE id='${id}';`
+
+        await executeDbRunSqlAsPromise(db, queryString);
+
+        logger.log();
+    }
+}
+
+// Transfers are a part of extrinsics table
+const convertTransfers = async (db, ss58Format) => {
+    const logger = new Logger('transfer');
+
+    const result = await executeDbRunSqlAsPromise(db, `select block_number, signer, args, method from extrinsic where method like 'transfer%'`);
+    const rows = result.rows;
+    logger.setTotalRows(rows.length);
+
+    for (let i = 0; i < rows.length; i++) {
+        const {block_number, signer, args, method} = rows[i];
+
+        const nextSigner = decode(signer, ss58Format);
+
+        let queryString = `UPDATE extrinsic SET signer='${nextSigner}'`
+
+        if (method === 'transfer' || method === 'transferKeepAlive' || method === 'transferAll') {
+            const [account] = JSON.parse(args);
+            const accountId = typeof account === 'string' ? account : account.id ? account.id : account.address20;
+
+            const nextAccountId = decode(accountId, ss58Format);
+
+            const nextArgs = args.replace(accountId, nextAccountId);
+            queryString += `, args='${nextArgs}'`;
+        }
+
+        queryString += ` WHERE block_number='${block_number}';`
+
+        await executeDbRunSqlAsPromise(db, queryString);
+
+        logger.log();
+    }
+};
+
+const convertTables = async (db, prefix) => {
+    await convertAccountTable(db, prefix);
+    await convertEraTables(db, prefix);
+    await convertRankingTable(db, prefix);
+    await convertFaucetTable(db, prefix);
+    await convertTransfers(db, prefix);
+}
+
+class Logger {
+    constructor(tableName) {
+        this.tableName = tableName;
+        this.totalRows = 0;
+        this.currentRow = 0;
+        console.log(`Migrating ${this.tableName}`);
+    }
+
+    setTotalRows = (totalRows) => {
+        this.totalRows = totalRows;
+        console.log(`Total Rows are ${this.tableName}`);
+    }
+
+    log = () => {
+        this.currentRow++;
+        const {currentRow, totalRows, tableName} = this;
+
+        if (currentRow === totalRows) {
+            console.log(`✅ Done for ${tableName} table`);
+        } else {
+            if (currentRow % 150 === 0) {
```

Comment:
Make less logs

Reply:
Done. 
<img width="514" alt="Screenshot 2023-03-22 at 14 13 22" src="https://user-images.githubusercontent.com/32759781/226872536-eee30c4d-a826-4595-819d-0b53693e80d5.png">

Reply:
Let's do step equal to `10`.